WeakAuras/RegionTypes/Icon.lua

```diff
@@ -69,7 +69,7 @@ local properties = {
     default = 32
   },
   glow = {
-    display = { WeakAuras.newFeatureString .. L["Glow"], L["Show Glow"] },
+    display = { WeakAuras.newFeatureString .. L["Glow"], L["Show Glow"], true },
```

Comment:
I dislike this for religious reasons, since display names don’t have anything to do with defaults. Is there any way that we can set the default flag on something other than the display name?

Reply:
Also some day in the distant future we may have a use for a dropdown tree that is three or more levels deep, and this interferes with that.

Reply:
i didn't find how to pass along more properties to the ace widget

Reply:
That isn't ace though, that's the property definitions which get converted to ace tables in ConditionOptions.lua. 

The code is pretty hard to follow though, and it's called display there too....